backend/windmill-api/openapi.yaml

```diff
@@ -6467,6 +6523,8 @@ components:
           type: number
         dedicated_worker:
           type: boolean
+        ws_error_handler_enabled:
+          type: boolean
```

Comment:
think it make more sense to store ws_error_handler_muted since by default it is enabled